mods/ra/rules/infantry.yaml

```diff
@@ -52,10 +52,14 @@ DOG:
 		MoveSequence: walk
 		StandSequences: stand
```

Comment:
Apparently this is already the default, which we could remove. On the other hand we have this in a lot of places (almost all WIB in the diff of this PR have it), so not an issue we need to solve in this PR imo.